### PR TITLE
Add new CSS handles

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?

Add new CSS handles to `minicart.v2`.

#### What problem is this solving?

There were some HTML tags that did not have CSS handles, so users would not be able to customize them.

#### How should this be manually tested?

[Minicart](https://minicart--storecomponents.myvtex.com)

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by iaronaraujo — APPROVED** [review]

**Review by kaisermann — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -1,195 +1,83 @@
-# VTEX Minicart
+📢 Use this project, [contribute](https://github.com/vtex-apps/breadcrumb) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```
Comment (klzns) [inline docs/README.md]: I think there's a lot of places with that link being wrong. I just noticed this yesterday.

```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/minicart) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

File: docs/README.md [hunk 2]
```diff
@@ -1,195 +1,83 @@
...
+## Contributors
 
 Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
 <!-- prettier-ignore -->
 <table>
   <tr>
-    <td align="center"><a href="https://github.com/lucasayb"><img src="https://avatars2.githubusercontent.com/u/17356081?v=4" width="100px;" alt="Lucas Ant�nio Yamamoto Borges"/><br /><sub><b>Lucas Ant�nio Yamamoto Borges</b></sub></a><br /><a href="https://github.com/vtex-apps/minicart/commits?author=lucasayb" title="Code">=�</a></td>
+    <td align="center"><a href="https://github.com/lucasayb"><img src="https://avatars2.githubusercontent.com/u/17356081?v=4" width="100px;" alt="Lucas Ant�nio Yamamoto Borges"/><br /><sub><b>Lucas Ant�nio Yamamoto Borges</b></sub></a><br /><a href="https://github.com/vtex-apps/minicart/commits?author=lucasayb" title="Code">=�</a></td>
```
Comment (klzns) [inline docs/README.md]: What happened here?

File: react/BaseContent.tsx [hunk 3]
```diff
@@ -15,15 +15,20 @@ interface Props {
...
+] as const
 
 const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
   const { orderForm, loading }: OrderFormContext = useOrderForm()
   const handles = useCssHandles(CSS_HANDLES)
   const { variation } = useMinicartState()
   const { url: checkoutUrl } = useCheckoutURL()
 
-  const minicartContentClasses = `${handles.minicartContent} ${
+  const minicartContentClasses = `${handles.minicartContentContainer} ${
```
Comment (klzns) [inline react/BaseContent.tsx:31]: Isn't that a breaking change?

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/BaseContent.tsx (hunk 3 above)
Comment (victorhmp) [reply react/BaseContent.tsx:31]: Not in this case, since nobody is using `minicart.v2` yet

File: docs/README.md (hunk 1 above)
Comment (victorhmp) [reply docs/README.md]: Woow! Good catch!

File: docs/README.md (hunk 2 above)
Comment (victorhmp) [reply docs/README.md]: Uuhm... I did not notice this

Comment (victorhmp) [reply docs/README.md]: Fixed this in the PR regarding documentation. I'll rebase this and it should be fixed :)

**Review by klzns — APPROVED** [review]

**Review by Klynger — APPROVED** [review]

**Pull request merged**